app/assets/stylesheets/active_admin.css

```diff
@@ -0,0 +1,7 @@
+@import 'tailwindcss' source(none);
+
+@utility ring-opacity-5 {
+  --tw-ring-opacity: 0.05;
+}
```

Comment:
This is to work around the fact that the ActiveAdmin Tailwind plugin tries to apply a `ring-opacity-5` that doesn't exist (anymore?). I think that this will be corrected in ActiveAdmin, maybe via this PR: https://github.com/activeadmin/activeadmin/pull/ 8649/files#diff-8a54c049b81d2642e951ea697e44aca5d74fb1b990552e8dd0383fe51917da95L436 .